sbin/start-slave.sh

```diff
@@ -19,7 +19,7 @@
 
 # Starts a slave on the machine this script is executed on.
 
-usage="Usage: start-slave.sh <worker#> <spark-master-URL> where <spark-master-URL> is like "spark://localhost:7077"
+usage="Usage: start-slave.sh <worker#> <spark-master-URL> where <spark-master-URL> is like "spark://localhost:7077""
```

Comment:
Ah I see, good catch. Actually, the quote in `"spark` should just be removed. I think. You can close your other PR and push an update to this one and I'll merge.